### PR TITLE
WFLY-14989 Inside naming remove unnecessary unboxing to clean code

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-14989

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@ropalka as we have discussed can you please approve this PR

File: naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java [hunk 1]
```diff
@@ -386,7 +386,7 @@ public String getJndiViewInstanceValue() {
             }
             ManagedReference reference = getReference();
             try {
-                return String.valueOf(reference.getInstance());
```
Comment (boris-unckel) [inline naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java:389]: This is not valid: String.valueOf(null) returns a String with the word "null". (String) null returns a null reference.

File: naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java [hunk 2]
```diff
@@ -431,7 +431,7 @@ public String getJndiViewInstanceValue() {
             final ClassLoader cl = WildFlySecurityManager.getCurrentContextClassLoaderPrivileged();
             try {
                 WildFlySecurityManager.setCurrentContextClassLoaderPrivileged(objectFactoryClassInstance.getClass().getClassLoader());
-                return String.valueOf(getReference().getInstance());
+                return (String) getReference().getInstance();
```
Comment (boris-unckel) [inline naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java:434]: This is not valid: String.valueOf(null) returns a String with the word "null". (String) null returns a null reference.

File: naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java [hunk 3]
```diff
@@ -234,7 +234,7 @@ public String getJndiViewInstanceValue() {
                 final ClassLoader cl = WildFlySecurityManager.getCurrentContextClassLoaderPrivileged();
                 try {
                     WildFlySecurityManager.setCurrentContextClassLoaderPrivileged(objectFactoryClassInstance.getClass().getClassLoader());
-                    return String.valueOf(getReference().getInstance());
+                    return (String) getReference().getInstance();
```
Comment (boris-unckel) [inline naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingAdd.java:237]: This is not valid: String.valueOf(null) returns a String with the word "null". (String) null returns a null reference.

File: naming/src/main/java/org/jboss/as/naming/management/JndiViewOperation.java [hunk 4]
```diff
@@ -148,7 +148,7 @@ private void addEntries(final ModelNode current, final Context context) throws N
                            NamingLogger.ROOT_LOGGER.failedToLookupJndiViewValue(pair.getName(),e);
                        }
                     } else if (!(value instanceof ManagedReferenceFactory)) {
-                       jndiViewValue = String.valueOf(value);
+                       jndiViewValue = (String) value;
```
Comment (boris-unckel) [inline naming/src/main/java/org/jboss/as/naming/management/JndiViewOperation.java:151]: This is not valid: String.valueOf(null) returns a String with the word "null". (String) null returns a null reference.

**Comment by RanabirChakraborty** [comment]
Thanks! @boris-unckel done the changes.

**Review by ropalka — APPROVED** [review]
LGTM

**Pull request merged**